mp/src/game/shared/neo/neo_gamerules.cpp

```diff
@@ -31,7 +31,7 @@ ConVar mp_neo_latespawn_max_time("mp_neo_latespawn_max_time", "15", FCVAR_REPLIC
 ConVar sv_neo_wep_dmg_modifier("sv_neo_wep_dmg_modifier", "0.5", FCVAR_REPLICATED, "Temp global weapon damage modifier.", true, 0.0, true, 100.0);
 
 ConVar neo_name("neo_name", "", FCVAR_USERINFO | FCVAR_ARCHIVE, "The nickname to set instead of the steam profile name.");
-ConVar cl_fakenick("cl_fakenick", "1", FCVAR_USERINFO | FCVAR_ARCHIVE, "Show players set neo_name, otherwise only show Steam names.", true, 0.0f, true, 1.0f);
+ConVar cl_fakenick("cl_onlysteamnick", "1", FCVAR_USERINFO | FCVAR_ARCHIVE, "Only show players Steam names, otherwise show player set names.", true, 0.0f, true, 1.0f);
```

Comment:
All `cl_fakenick` in code both variable and in comments should change to `cl_onlysteamnick` also. Also change comment a bit EX: L1375 neo_gamerules.cpp `(only used if cl_fakenick=1` to `(only used if cl_onlysteamnick=0`

Also I think `cl_onlysteamnick` should be off/0 like how cl_fakenick=1 was before.